book/src/SUMMARY.md

```diff
@@ -28,5 +28,7 @@
 
 # Advanced Usage
 
-- [SDK](./advanced-usage/sdk.md)
+- [SDK](./advanced-usage/sdk/index.md)
+  - [App Proof Generation and Verification](./advanced-usage/sdk/app-proof.md)
```

Comment:
have you previewed this with `mdbook serve --open`? this makes a sidebar item that is really long